### ISSUE TITLE
Skip with where and order

### ISSUE THREAD

**Issue by Pyreweb** [ujjwalguptaofficial/JsStore#132]
Hi, 
When using the transaction controls 'skip' ,'where' and 'order', the results are incorrect.

example: https://ujjwalguptaofficial.github.io/idbstudio/?db=Demo
```
select({
    from: "Customers",
    limit: 10,
    order: {
        by: 'address',
        type: "asc"
    },
    where: {
        customerId: {
            '>': 10
        }
    },
    skip: 20
});
```
In exemple, the first line (customer id 41) is already in return data until skip 31.

**Comment by ujjwalguptaofficial** [comment]
Issue fixed in v - 3.4.1. Please check and let me know.

**Comment by Pyreweb** [comment]
That's ok. Thank you.
